.env

```diff
@@ -0,0 +1,5 @@
+DB_USER=api_user
+DB_PASSWORD=password
+DB_HOST=localhost
+DB_PORT=5432
+DB_DATABASE=songs
```

Comment:
I'm really curious about the syntax for this .env file. We should look into it later!